src/main/java/com/teks/users/user/model/User.java

```diff
@@ -1,33 +1,46 @@
 package com.teks.users.user.model;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
+import javax.persistence.*;
 
 @Entity
 public class User {
-    @Id
-    @GeneratedValue
-    private int id;
-
+//    @Id
+//    @GeneratedValue(strategy = GenerationType.AUTO)
+//    private int id;
```

Comment:
Why did you remove this field? You don't have a primary key in your database table!